streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -810,6 +804,7 @@ public void run() {
      * Shutdown this {@code KafkaStreams} instance by signaling all the threads to stop, and then wait for them to join.
      * This will block until all threads have stopped.
      */
+    @SuppressWarnings("deprecation")
```

Comment:
Because we call deprecated method `close(DEFAULT_CLOSE_TIMEOUT, TimeUnit.SECONDS);` -- this method will stay though, ie, we will make it private when we "remove" it from public API and than we can remove this annotation, too.